yearn/test/system/convex-strategy.test.js

```diff
@@ -220,5 +221,24 @@ describeFn("System -- convex strategy", () => {
       expect(amountWithdrawn.gt(curveStrategyFixture.vaultDepositAmount)).to.be
         .true
     })
+
+    describe("eth to LP token (want) conversion", () => {
+      // At block 12786839 (Jul-08-2021) the exchange rate between wBTC/ETH was
+      // ~1/15.65 Swapping 1000ETH on Uniswap gave ~63.9wBTC. It means that by
+      // depositing 63.9wBTC to Curve.fi LP Pool a depositor would get
+      // ~63,3018 LP tokens.
```

Comment:
Can we use a dot instead of a comma? I thought we're talking about thousands at the first sight.

Reply:
Sure e369c05